src/current/v24.1/set-up-physical-cluster-replication.md

```diff
@@ -99,25 +87,31 @@ Connect to your primary cluster's system virtual cluster using [`cockroach sql`]
     SET CLUSTER SETTING enterprise.license = 'your enterprise license';
     ~~~
 
+1. Set the `kv.rangefeed.enabled` cluster setting to `true`. The replication job connects to a long-lived request, a _rangefeed_, which pushes changes as they happen:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING kv.rangefeed.enabled = true;
+    ~~~
+
 1. Confirm the status of your virtual cluster: {% comment %}Link to SQL ref page here{% endcomment %}
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
     SHOW VIRTUAL CLUSTERS;
     ~~~
 
-    The output will include the `system` interface and the `application` virtual cluster:
+    The output will include the `system` virtual cluster and the `main` virtual cluster:
 
     ~~~
-    id |    name     | data_state | service_mode
-    ---+-------------+------------+---------------
-    1  | system      | ready      | shared
-    2  | template    | ready      | none
-    3  | application | ready      | shared
-    (3 rows)
+      id |  name  | data_state | service_mode
+    -----+--------+------------+---------------
+       1 | system | ready      | shared
+       3 | main   | ready      | shared
```

Comment:
I don't know if this really matters, but the newly created `main` default VC always creates at `id 3`... which seemed like an artifact of the `template` VC being there.

Reply:
We do this on purpose because we _may_ want to use `id 2` for something special in the future.

Reply:
Ooooo ok! That sounds interesting, can't wait to find out what it is!! 😄 🍿